cmis/src/main/java/org/frankframework/extensions/cmis/CmisSender.java

```diff
@@ -1203,8 +1188,26 @@ public void setProxyAuthAlias(String proxyAuthAlias) {
 	public void setProxyUsername(String proxyUsername) {
 		sessionBuilder.setProxyUsername(proxyUsername);
 	}
+
 	/** Proxy Password */
 	public void setProxyPassword(String proxyPassword) {
 		sessionBuilder.setProxyPassword(proxyPassword);
 	}
+
+	public enum CmisAction {
+		/** Create a document */
+		CREATE,
+		/** Delete a document */
+		DELETE,
+		/** Get the content of a document (and optional the properties) */
+		GET,
+		/** Perform a query that returns properties */
+		FIND,
+		/** Update the properties of an existing document */
+		UPDATE,
+		/** Get the (meta)data of a folder or document */
+		FETCH,
+		/** Determine action based on the incoming CmisEvent */
+		DYNAMIC
+	}
```

Comment:
Ik vind het wel iets wat onduidelijk als deze onderaan staat...

Reply:
Dat is volgens de formatting uit het project. Ik heb formatting voor nu even gerevert want het raakt op sommige plekken teveel (zoals deze file)